frontend/tailwind.config.js

```diff
@@ -91,8 +104,15 @@ module.exports = {
   			lg: 'var(--radius)',
   			md: 'calc(var(--radius) - 2px)',
   			sm: 'calc(var(--radius) - 4px)'
-  		}
-  	}
-  },
-  plugins: [require("tailwindcss-animate")],
+  		},
+		screens: {
+			sm: '640px',
+			md: '768px',
+			lg: '1024px',
+		},
+	  },
+	},
+	plugins: [],
 };
+
+//   plugins: [require("tailwindcss-animate")],
```

Comment:
👀

Reply:
This line is from another teammate, but it cause error and he didn't need it anymore. But I'd like to leave it here for now in case he might want it back later.